components/Common/ProgressionSidebar/index.module.css

```diff
@@ -0,0 +1,14 @@
+.wrapper {
+  @apply flex
+    h-[calc(100vh-var(--header-height,2rem))]
```

Comment:
Where dies this variable come from?

Reply:
I removed it. It will be better to adjust the height when creating new layouts